### PR TITLE
tests/kola/selinux: add test that policy isn't recompiled

### PR DESCRIPTION
**Pull request by jlebon**
This is a test for https://github.com/openshift/os/issues/1036. It also exists in the rpm-ostree CI, but let's have it here too since other packages can break this.

### CODE REVIEW

**Comment by jlebon** [comment]
Let's hold this until [the rpm-ostree patch](https://github.com/coreos/rpm-ostree/pull/4122) is in f37.

**Comment by jlebon** [comment]
Actually, all we need is for the rpm-ostree patch to get into cosa, which it is now. So this is ready for review!

File: tests/kola/selinux/default [hunk 1]
```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
+# kola
+#   exclusive: false
+#   # if it works on QEMU, it should work everywhere
+#   platforms: qemu
```
Comment (dustymabe) [inline tests/kola/selinux/default]:
```suggestion
## kola:
##   exclusive: false
##   tags: "platform-independent"
```

Comment (jlebon) [reply tests/kola/selinux/default]: Done!

Comment (cgwalters) [reply tests/kola/selinux/default]: Heh, we may need some sort of test for the test syntax...

File: tests/kola/selinux/default [hunk 2]
```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
+# kola
+#   exclusive: false
+#   tags: "platform-independent"
```
Comment (dustymabe) [inline tests/kola/selinux/default]: I think we still need some updates here. IIUC we need two `##` at the beginning of the line and the first line needs to be `## kola:` (with a `:`). Am I missing something?

```suggestion
## kola:
##   exclusive: false
##   tags: "platform-independent"
```

Comment (dustymabe) [reply tests/kola/selinux/default]: may also be nice to get a  description of the problem and maybe a link to https://github.com/openshift/os/issues/1036?

I also wonder if we shouldn't have an FCOS issue tracker issue for this problem.

Comment (jlebon) [reply tests/kola/selinux/default]: OK, I followed the format used in the one other test that uses `platform-independent`.

Comment (dustymabe) [reply tests/kola/selinux/default]: You mean this one:

https://github.com/coreos/fedora-coreos-config/blob/010ad3d732241c8f2eb19133511c325d34503b52/tests/kola/content-origins/test.sh#L3-L8

File: tests/kola/selinux/default [hunk 3]
```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
+# Verify that the SELinux policy isn't marked as modified.
+# See: https://github.com/openshift/os/issues/1036
+## kola
```
Comment (dustymabe) [inline tests/kola/selinux/default]:
```suggestion
## kola:
```

Is this not needed?

File: tests/kola/selinux/default (hunk 2 above)
Comment (jlebon) [reply tests/kola/selinux/default]: Added link back to https://github.com/openshift/os/issues/1036. Not sure it's worth the FCOS issue tracker at this point. Were you thinking for release notes?

**Review by dustymabe — APPROVED** [review]
LGTM - thanks for writing this test!

File: tests/kola/selinux/default (hunk 2 above)
Comment (dustymabe) [reply tests/kola/selinux/default]: Mostly just for a record that this affected `next` stream nodes. I'm not too worried about it, though. Fine without it.

**Pull request merged**